src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -33,7 +33,7 @@ private void addCategory() {
                 item.setCategory(category);
             }
         } catch (NullPointerException e) {
-            item.setCategory("Uncategorized");
```

Comment:
I think might need check the ug again for the this change as I thought we are going to use "Uncategorized" originally

Reply:
Because of the `.toLowerCase()` function in most of the updating/modifying category codes, even if the input was initialized to 'Uncategorized' it will end up printing as 'uncategorized' instead in list function after a few adding/editing/removing move has been done. Also, some of the tests with had 'Uncategorized' did not manage to pass. Might be more ideal to standardize to all lowercases.